packages/presentation/lib/screen/home/home_view_mapper.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:domain/model/movie_response_model.dart';
+import 'package:domain/utils/extensions/string_extension.dart';
+import 'package:domain/utils/extensions/int_extension.dart';
+import 'package:domain/utils/getSecrets.dart';
+import 'package:yaml/yaml.dart';
+import 'movie_model.dart';
+
+abstract class HomeViewMapper {
+  factory HomeViewMapper() => _HomeViewMapper();
+
+  Future<List<MoviePresentation>> mapMovieDataToRequest(
+      List<MovieResponse> response);
+}
+
+class _HomeViewMapper implements HomeViewMapper {
+  @override
+  Future<List<MoviePresentation>> mapMovieDataToRequest(
+    List<MovieResponse> response,
+  ) async {
+    String key = await getImageSecret();
```

Comment:
-//-

Reply:
Can not understand what is this related for, could you please specify